qiskit_experiments/test/pulse_backend.py

```diff
@@ -591,5 +613,220 @@ def __init__(
             self._defaults.instruction_schedule_map.get("sx", (0,)),
         ]
         self._simulated_pulse_unitaries = {
-            (schedule.name, (0,), ()): self.solve(schedule, (0,)) for schedule in default_schedules
+            (schedule.name, (0,), ()): self.solve(schedule) for schedule in default_schedules
         }
+
+    @staticmethod
+    def rz_gate(_, theta: float) -> np.ndarray:
+        """Rz gate corresponding to single qubit 3 level qubit. Note: We do not try to
+        model accurate qutrit dynamics.
+
+        Args:
+            theta: The angle parameter of Rz gate.
+
+        Returns:
+            Matrix of Rz(theta).
+        """
+        return np.diag([np.exp(1.0j * idx * theta / 2) for idx in [-1, 1, 3]])
+
+
+class ParallelTransmonTestBackend(PulseBackend):
+    r"""A backend that corresponds to a three level anharmonic transmon qubit.
+
+    The Hamiltonian of the system is
+
+    .. math::
+        H^i = \hbar \sum_{j=1,2} \left[\omega^i_j |j\rangle\langle j| +
+                \mathcal{E}(t) \lambda^i_j (\sigma_j^+ + \sigma_j^-)\right]
+
+        H = H^0 ⊗ I + I ⊗ H^1
+    Here, :math:`\omega^i_j` is the ith transition frequency from level :math`0` to level
+    :math:`j`. :math:`\mathcal{E}(t)` is the drive field and :math:`\sigma_j^\pm` are
+    the raising and lowering operators between levels :math:`j-1` and :math:`j`.
+    """
+
+    def __init__(
+        self,
+        qubit_frequency: float = 5e9,
```

Comment:
I'm also against to having user provided frequency and anharmonicity. Because this changes the Hamiltonian, you need to dynamically update the hard-coded calibrations. For example, currently DRAG pulse is not considered in the calibration but changing anharmonicity may change DRAG coefficient. The missing DRAG term may impact the unittest of the experiments requiring precise state preparation or echo, such as error amplification.

I prefer to have DRAG pulses and remove frequencies and anharmonicities from the constructor to preserve the hard-coded calibrations.